languages/ruby/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,53 @@
+Ruby is a dynamic [object-oriented language]. Everything in Ruby is an [object].
+
+There are two ways to assign objects to names in Ruby - assigning variables or constants. Variables always start with lower-case letters and use "snake case" for their formatting. A variable can be redefined to different objects over its lifetime. For example, `my_first_variable` can be defined and redefined many times using the `=` operator:
```

Comment:
Maybe define the snake casing in the description itself and link to the [Snake case Wikipedia page](https://en.wikipedia.org/wiki/Snake_case)?

```suggestion
There are two ways to assign objects to names in Ruby - assigning variables or constants. Variables always start with lower-case letters and use [snake_case][snake_case] for their formatting. A variable can be redefined to different objects over its lifetime. For example, `my_first_variable` can be defined and redefined many times using the `=` operator:
```